### ISSUE TITLE
How to get the cosine similarity eq2 from the function att_emb_ngram_encoder_maxout?  

### ISSUE THREAD

**Issue by tongcu** [guoyinwang/LEAM#16]
How to get the cosine similarity(eq2) from the function att_emb_ngram_encoder_maxout? 
AND what is mean of the following code Att_v = tf.contrib.layers.conv2d(G, num_outputs=opt.num_class,kernel_size=[opt.ngram], padding='SAME',activation_fn=tf.nn.relu) #b * s *  c in paper?
